docs/content.zh/docs/deployment/elastic_scaling.md

```diff
@@ -23,134 +23,132 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-# Elastic Scaling
+# 弹性伸缩
 
-Apache Flink allows you to rescale your jobs. You can do this manually by stopping the job and restarting from the savepoint created during shutdown with a different parallelism.
+在 Apache Flink 中，可以通过手动停止 Job，然后从停止时创建的 Savepoint 恢复，最后重新指定并行度的方式来重新伸缩 Job。
 
-This page describes options where Flink automatically adjusts the parallelism instead.
+这个文档描述的特性是 Flink 如何自动地调整并行度。
```

Comment:
I think 'options' might not be correctly translated. `options` means users can choose either reactive mode or adaptive scheduler.

Reply:
Here is the revised translation:
https://github.com/apache/flink/pull/16928/commits/809752c45be82684c3dd51553db9ee6ed5807f00#diff-69b2dca8ca68dd36524d019124dd2eef1e7914f45f6992733e7d221c13b68ea9R30